app/Models/Concerns/Filterable.php

```diff
@@ -315,7 +343,7 @@ protected function extractSearchOperator($value, string $key): array
     {
         $operator = 'like';
         $filterValue = $value;
-        if ($key !== 'tags') {
+        if (($key !== 'tags') && ($key !== 'races')) {
```

Comment:
Is this change actually needed?

Reply:
Yes, if that change is not made the following error message is displayed when trying to search using races as a parameter:

`strncmp(): Argument #1 ($string1) must be of type string, array given`